.gitignore

```diff
@@ -2,6 +2,9 @@
 *~
 *#*
 
+# vim temporary files
+.*.sw*
```

Comment:
This pattern should probably be: `*.sw[a-p]`. It isn't matching some swap files I have locally.

Reply:
Interesting. Both work on my machine, and I'm happy to change it.

I'm curious which swap files it wasn't picking up locally, just because I'm wondering what the bug in `.*.sw*` is, but it's a totally academic interest.

Reply:
It doesn't match `.swp` or `.swo`. I think the more completely correct gitignore for swap files can be found here: https://github.com/github/gitignore/blob/master/Global/Vim.gitignore. The pattern I provided seems to be a common pattern used for swap files in gitignore.

Reply:
Thanks!